app/controllers/provider_interface/content_controller.rb

```diff
@@ -0,0 +1,31 @@
+module ProviderInterface
+  class ContentController < ActionController::Base
```

Comment:
Would ideally inherit from `ProviderInterfaceController` here, but that forces authentication, and you don’t need to be signed in to view policies. I pulled across the relevant parts of that controller to use here, but open to suggestions if better way to achieve this.

Reply:
You can inherit from `ProviderInterfaceController` and add:

```rb
skip_before_action :authenticate_provider_user!
```

Reply:
Oooooh 😀